xbmc/utils/Mime.cpp

```diff
@@ -486,10 +486,11 @@ static std::map<std::string, std::string> fillMimeTypes()
   mimeTypes.insert(std::pair<std::string, std::string>("xlw",       "application/excel"));
   mimeTypes.insert(std::pair<std::string, std::string>("xm",        "audio/xm"));
   mimeTypes.insert(std::pair<std::string, std::string>("xml",       "text/xml"));
+  mimeTypes.insert(std::pair<std::string, std::string>("x-png",     "image/png"));
   mimeTypes.insert(std::pair<std::string, std::string>("xmz",       "xgl/movie"));
   mimeTypes.insert(std::pair<std::string, std::string>("xpix",      "application/x-vnd.ls-xpix"));
   mimeTypes.insert(std::pair<std::string, std::string>("xpm",       "image/xpm"));
-  mimeTypes.insert(std::pair<std::string, std::string>("x-png",     "image/png"));
```

Comment:
Why is this line moved?

Reply:
Artifact of sort, it actually should have moved higher but I missed I hadn't selected all of the 'x' range. Looking at it again, though, it does make sense to be at `xp` despite `x-p` lexically sorting before `xi`.

Reply:
Artifact of sort; it actually should have moved higher but I missed I hadn't selected all of the `x` range. Looking at it again, though, it does make sense to be at `xp` despite `x-p` lexically sorting before `xi`.